core/jobs/batch_jobs/test_gcs_io_job.py

```diff
@@ -0,0 +1,100 @@
+# coding: utf-8
+#
+# Copyright 2022 The Oppia Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS-IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Tests the functionality of gcs_io."""
+
+from __future__ import annotations
+
+from core.jobs import base_jobs
+from core.jobs.io import gcs_io
+from core.jobs.transforms import job_result_transforms
+from core.jobs.types import job_run_result
+from core.platform import models
+
+import apache_beam as beam
+from apache_beam.io.gcp import gcsio_test
+from typing import Optional
+
+datastore_services = models.Registry.import_datastore_services()
+
+
+class TestGCSIoJob(base_jobs.JobBase):
```

Comment:
I think we should have a separate job for each IO so that we can run, see if the files exist through Google Cloud Console, and then run the second job for reading the files.

Reply:
Done